db/seeds.rb

```diff
@@ -148,15 +148,15 @@
       {
         # Canvas
         iss: "https://canvas.instructure.com",
-        client_id: "43460000000000378",
+        client_id: secrets.canvas_developer_id,
```

Comment:
Did you add a new value to the secrets? This value is the id Canvas provides when you setup a new LTI Advantage app. Make sure this isn't the developer id associated with the OAuth key and secret.

Reply:
![image](https://user-images.githubusercontent.com/24883544/101377787-86dba900-386f-11eb-81c0-735e1703604a.png)
Aren't these equivalent? The README said to put the ID and Key provided by canvas into your .env file, with the ID being the same value we use when installing an LTI Advantage app. Unless I'm misunderstanding, those two should be equivalent?

Reply:
They are different. The one in this file is used as the id for the LTI Advantage application. The other is used to let a user do the OAuth dance with our application so we can get a Canvas API keyl